### PR TITLE
How to install latest CMake if required

### PR DESCRIPTION
**Pull request by amilcarlucas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,20 @@ If you do not have an ARM compiler you will need to install one:
 sudo apt install gcc-arm-none-eabi
 ```
 
-This component needs `CMake` >= 3.19, `colcon` and other Python 3 packages in order to build micro-ROS packages:
+This component needs `CMake` >= 3.19.
+If required, you can install the latest cmake version with the following commands:
+
+```bash
```
Comment (pablogs9) [inline README.md:16]: I'm not sure if we should include this.
Which of those packages are Mbed dependencies? In that case, we should leverage Mbed instructions in order to not provide outdated information.

Regarding CMake, do Mbed provide instructions for updating CMake to the required version? If not, we should link to Kitware official instructions, also in order to not provide outdated instructions.

**Review by pablogs9 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (amilcarlucas) [reply README.md:16]: All packages are `mbed` dependencies. I did not find `CMake` instructions on the mbed website. I got these exact instructions from @Acuadros95

Comment (pablogs9) [reply README.md:16]: If they are mbed deps we should link the mbed installation instructions.

**Comment by amilcarlucas** [comment]
Linked to upstream

**Comment by pablogs9** [comment]
@mergify backport main

**Pull request merged**